environments/swiss/meta.yml

```diff
@@ -3,4 +3,5 @@ env_monitoring_id: swiss
 users:
   - dimagi
   - fissionlabs
+  - gcastro
```

Comment:
I think you need to add this user to a `.yml` file like this one:

https://github.com/dimagi/commcare-cloud/blob/master/environments/_users/dimagi.yml#L1

Reply:
+1